### ISSUE TITLE
about model result

### ISSUE THREAD

**Issue by LiUzHiAn** [rajatthepagal/RDNSR-Residual-Dense-Network-for-Super-Resolution-Keras#1]
Hi, rajatkb

Can I know your test result based on the model you have trained ? Specificlly, what is the PSNR value on Set5 test set (or other benchmark dataset ) of x2 scale .
 
Thank you

**Comment by rajatkb** [comment]
It's been a year I have worked on it so I don't remember about the PSNR values but for Set5 at 2x they we really good, I remember that (almost the same as given by many networks at 2x). Still, you can always calculate the PSNR by the results posted here. 
You can also check out here there are more results (bad and good to give idea about when the model may fail)

https://github.com/rajatkb/Deep-Super-Resolution-Research/tree/master/RDNSRnet
